contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,516 @@
+  Appropriately used as scratch space, it does not have the HostPath, became an initial offering for local storage, but had many
+  drawbacks. Without having the ability to guarantee space & ensure ownership, one would lose data once a node was rescheduled. 
+  Therefore, the risk outweighed the reward when trying to  leverage the power of local storage needed for stateful applications like 
+  databases.
+  
+  By extending the API for volumes to specifically request a raw block device, we provide an explicit method for volume comsumption,
+  whereas previously it was always a fileystem. In addition, the ability to use a raw block device without a filesystem will allow
+  Kubernetes better support of high performance applications that can utilize raw block devices directly for their storage. 
+  For example, MariaDB or MongoDB.
+  
+  Specific use cases around improved usage of storage consumption are included in the use cases listed below as follows:
+  * An admin wishes to expose a block volume to be consumed as a block volume for the user
+  * A user wishes to be specific about the storage consumption and intentionally request a block device
+  * A user wishes to utilitze block storage to fully realize the performance of an application tuned to using block devices
+  * A user wishes to utilize raw block devices for consumption from a virtual machine
+  Future use cases include dynamically provisioning and intelligent discovery of existing devices, which this proposal sets the foundation
+  for more fully developing these methods to enable block volume consumption.
+  
+    
+# Design Overview
+
+  The proposed design is based on the idea of leveraging well defined concepts for storage in Kubernetes. The consumption and 
+  definitions for the block devices will be driven through the PVC and PV and Storage Class definitions. Along with Storage
+  Resource definitions, this will provide the admin with a consistent way of managing all storage. 
+  The API changes proposed in the following section are minimal with the idea of defining a volumeType to indicate both the definition
+  and consumption of the devices. Since it's possible to create a volume as a block device and then later consume it by provisioning
+  a filesystem on top, the design requires explicit intent for how the volume will be used.
+  The additional benefit of explicitly defining how the volume is to be consumed will provide a means for indicating the method
+  by which the device should be scrubbed when the claim is deleted, as this method will differ from a raw block device compared to a 
+  filesystem. The ownership and responsibility of scrubbing the device properly shall be up to the plugin method being utilized. 
+  By explicitly having volumeType in the PV and PVC can help the storage provider determine what scrubbing method to use. As an example,
+  for local storage:
+  
+  | PV                | PVC              | action              |  
```

Comment:
We do not want to anticipate how plugins should clean up their fs..  its really up to the plugin vendor how the retention policy is executed...  This should move to local storage proposal.